File: code/ddmail_parent/ddmail_sellergoods/ddmail_sellergoods_interface/src/main/java/cn/xmo/ddmail/sellergoods/service/BrandService.java
package cn.xmo.ddmail.sellergoods.service;

import cn.xmo.ddmail.pojo.TbBrand;
import entity.PageResult;

import java.util.List;

/**
 * @Author: Elves
 * @Description:
 * @Date: Created in 20:52 2019/7/19
 * @Modified By:
 */
public interface BrandService {

     List<TbBrand> findAll();

     PageResult<TbBrand> findPage(int page, int size);

    void save(TbBrand brand);

    TbBrand findOne(Long id);

    void delete(String ids);
}

File: code/ddmail_parent/ddmail_manager_web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    this.findAll = function () {
        return $http.get('../brand/findAll');
    };
    this.findPage = function (page, size) {
        return $http.get('../brand/findPage/'+page+"/"+size);
    };
    this.save = function (entity) {
        return $http.post('../brand/save', entity);
    };
    this.findOne = function (id) {
        return $http.get('../brand/findOne/'+id);
    };
    this.delete = function (selectIds) {
        return $http.get('../brand/delete/'+selectIds );
    }
});